.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
I think this doesn't actually use containers. You might want to change the commit message to remove that reference or actually use containers by specifying `sudo: false`: https://blog.travis-ci.com/2016-11-08-trusty-container-public-beta.